android/src/com/unciv/app/PlatformSpecificHelpersAndroid.kt

```diff
@@ -35,6 +35,7 @@ Sources for Info about current orientation in case need:
         if (activity.requestedOrientation != orientation) activity.requestedOrientation = orientation
     }
 
+    @Suppress("DEPRECATION")
```

Comment:
That won't help your Feng Shui much, as the code will FAIL on Pie and below. I may have done something like
```kotlin
    override fun hasDisplayCutout() = when {
        Build.VERSION.SDK_INT >= Build.VERSION_CODES.R ->
            activity.display?.cutout != null
        Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q ->
            @Suppress("DEPRECATION")
            activity.windowManager.defaultDisplay.cutout != null
        Build.VERSION.SDK_INT >= Build.VERSION_CODES.P ->
            true  //todo G00gle tells us cutouts work on Pie but not how to determine whether there are any
        else -> false
    }
```
..and have _loudly_ cursed the API devs (Long.MAX_VALUE)-fold for f(/!"&(!ing that API up so convolutedly. With this, you would enable the cutout toggle on Pie, but it would do nothing if there isn't any? Probably. Or remove the P->true to just not support cutouts on Pie. Note the R branch doesn't do more or better, it just would help the future dev that wishes to raise minApi to 30.

Reply:
> `activity.display?.cutout != null`

the SDK code is broken. That function returns null if the phone is rotated to the right and works in the rest of the cases :/

Reply:
P.S.: For Pie, a `ViewCompat.getRootWindowInsets(View(activity))?.displayCutout != null` just might work - or not. Just a wild guess. It would create and right after GC a dummy view for the activity instead of using the actual one, which I have no idea how to get hold of. But WindowInsetsCompat.displayCutout seems to be the correct and only available approach, and ViewCompat.getRootWindowInsets "sounds like" it would be a viable way to get hold of a valid instance of WindowInsetsCompat (the official docs are maximally unhelpful on that - evil intent couldn't have produced a less useful doc).

Reply:
the problem is that `WindowInsetsCompat.displayCutout` always null regardless if you have a cutout or not. 
I tried with `WindowInsetsCompat.CONSUMED.displayCutout != null` but it just returns false

Reply:
In that case I defer to your better expertise.

I guess WindowInsetsCompat instances can be obtained from different sources, and one of them may do the cutouts right - most WindowInsetsCompat samples out there get theirs through an override, but that one lives on the ViewCompat _instance_ which I wouldn't know where to place. Android native UI API model requires some serious brain defect to understand. And while my brain _is_ damaged (or I wouldn't waste time here), the damage obviously doesn't meet requirements.